sevenseconds/config/vpc.py

```diff
@@ -76,10 +81,15 @@ def configure_vpc(account, region, base_ami_id):
             with ActionOnExit('Deleting subnet {subnet_id}..', subnet_id=subnet.id):
                 if not account.dry_run:
                     subnet.delete()
-    for _type in 'dmz', 'internal':
-        for i, az in enumerate(sorted(availability_zones)):
-            net = calculate_subnet(vpc_net, _type, i)
-            configure_subnet(vpc, az, _type, net, account.dry_run, ec2c.get_waiter('subnet_available'))
+
+    # Configure subnets
+    if vpc_config and 'subnets' in vpc_config:
```

Comment:
nit (not knowing python too much): on line 25, it is check as vpc_config is not None, here it is just checked less explicitly

Reply:
It doesn't really make sense now that I think about it. Fixed, thanks!